cli/pkg/kctrl/cmd/version.go

```diff
@@ -26,11 +38,42 @@ func NewVersionCmd(o *VersionOptions, flagsFactory cmdcore.FlagsFactory) *cobra.
 		Short: "Print client version",
 		RunE:  func(_ *cobra.Command, _ []string) error { return o.Run() },
 	}
+
+	cmd.Flags().BoolVar(&o.controllerVersion, "controller", false, "Get the version of kapp-controller deployed on the cluster")
+
 	return cmd
 }
 
 func (o *VersionOptions) Run() error {
+	if o.controllerVersion {
+		err := o.showControllerVersion()
+		if err != nil {
+			return err
+		}
+		return nil
+	}
+
 	o.ui.PrintBlock([]byte(fmt.Sprintf("kctrl version %s\n", version.Version)))
 
 	return nil
 }
+
+func (o *VersionOptions) showControllerVersion() error {
+	coreClient, err := o.depsFactory.CoreClient()
+	if err != nil {
+		return err
+	}
+
+	controllerDeployment, err := coreClient.AppsV1().Deployments(kappControllerNamespace).Get(context.Background(), kappControllerDeployment, metav1.GetOptions{})
+	if err != nil {
+		if errors.IsNotFound(err) {
+			return fmt.Errorf("kapp-controller not installed on cluster")
```

Comment:
nit: `kapp-controller not installed on cluster (or is not present in the default location namespace=kapp-controller deployment=kapp-controller)`

Reply:
Done